src/tests/java/htsjdk/variant/vcf/VCFHeaderUnitTest.java

```diff
@@ -61,6 +69,19 @@ private VCFHeader createHeader(String headerStr) {
         return header;
     }
 
+    @BeforeClass
+    private void createTemporaryDirectory() {
```

Comment:
Fine as long as in the GATK we still have a method in `BaseTest` that calls into the underlying utility, wherever it may live.